### ISSUE TITLE
Refresh style typography for Figma

### ISSUE THREAD

**Issue by AndySobol** [six7/figma-tokens#1183]
@six7 Hello!
**Is your request related to the problem? Describe.**
I have a problem updating typography style in Figma itself. 

**Describe the required solution**
I would like to know (maybe there is a plugin) how it is possible to update the text styles that I registered in Figma if I changed the values of tokens (for example the size of the font) and got them using GitHub in the plugin Figma Tokens. In the video I show you visually. 
[Video problems for YouTube](https://youtu.be/mnWSOC3q73A)

**Describe the alternatives considered**
I thought that the text styles would update themselves as the style names were saved, only the values changed. But the Styles just fall apart and the update does not occur =( There may be a plugin, but after trying several options, did not find the right one.

**Additional context**
I would be grateful for any help. And thank you so much for your work!

**Comment by six7** [comment]
This sounds similar to #1194 - would that solve it for you?

**Comment by six7** [comment]
This sounds similar to #1194 - would that solve it for you?

**Comment by six7** [comment]
This sounds similar to #1194 - would that solve it for you?

**Comment by six7** [comment]
This sounds similar to #1194 - would that solve it for you?

**Comment by six7** [comment]
This sounds similar to #1194 - would that solve it for you?

**Comment by AndySobol** [comment]
I understood the mechanics of updating styles. Explain what I have: 

I have a Core group - it’s a dimensional grid font. 

And from this grid, I create typography. 
Let’s say for headline I use the following entry:

`headline = core.x20-fontSizes`

The **headline** is a typography token. 
**core.x20-fontSizes** - the font size from the grid fonts. 

So, headline having **type=typography**, updates styles and works well. But the core is not updated, which I also added in Figma styles. 

I originally wanted to have fixed styles of font size (core) and typography in my figma, but when I realized that this was not possible and I didn’t need it, let the mathematics stay in the code instead of the fig.

Therefore, the matter could be closed. 

Thanks @six7 ❤️
